src/pages/Transaction/index.js

```diff
@@ -91,38 +91,41 @@ class Transaction extends React.Component {
 									<td>Fee</td>
 									<td>{this.state.loading ? <Load /> : (
 										<div className="tx-hasicon">
-											{formatValue(this.state.transaction.fee / 1000000)}
+											{formatValue(this.state.transaction.transaction.fee / 1000000)}
 											<AlgoIcon />
 										</div>
 									)}</td>
 								</tr>
 								<tr>
 									<td>First round</td>
-									<td>{this.state.loading ? <Load /> : <NavLink to={`/block/${this.state.transaction["first-round"]}`}>{this.state.transaction["first-round"]}</NavLink>}</td>
+									<td>{this.state.loading ? <Load /> : <NavLink to={`/block/${this.state.transaction.transaction["first-valid"]}`}>{this.state.transaction.transaction["first-valid"]}</NavLink>}</td>
 								</tr>
 								<tr>
 									<td>Last round</td>
-									<td>{this.state.loading ? <Load /> : <NavLink to={`/block/${this.state.transaction["last-round"]}`}>{this.state.transaction["last-round"]}</NavLink>}</td>
+									<td>{this.state.loading ? <Load /> : <NavLink to={`/block/${this.state.transaction.transaction["last-valid"]}`}>{this.state.transaction.transaction["last-valid"]}</NavLink>}</td>
 								</tr>
 								<tr>
 									<td>Timestamp</td>
 									<td>{this.state.loading ? <Load /> : moment.unix(this.state.transaction.timestamp).format("LLLL")}</td>
 								</tr>
 								<tr>
-									<td>Base 64</td>
+									<td>Note</td>
 									<td>
 										{this.state.loading ? <Load /> : (
 											<div>
-												{this.state.transaction.noteb64 && this.state.transaction.noteb64 !== '' ? (
-													<div>
-														<div>
-															<span>Base 64:</span>
-															<textarea defaultValue={this.state.transaction.noteb64} readOnly></textarea>
-														</div>
-														<div>
-															<span>Converted:</span>
-															<textarea defaultValue={atob(this.state.transaction.noteb64)} readOnly></textarea>
-														</div>
+												{this.state.transaction.transaction.note && this.state.transaction.transaction.note !== '' ? (
+													// <div>
+													// 	<div>
+													// 		<span>Base 64:</span>
+													// 		<textarea defaultValue={this.state.transaction.transaction.note} readOnly></textarea>
+													// 	</div>
+													// 	<div>
+													// 		<span>Converted:</span>
+													// 		<textarea defaultValue={atob(this.state.transaction.transaction.note)} readOnly></textarea>
+													// 	</div>
+													// </div>
+                                                    <div>
+														<textarea defaultValue={this.state.transaction.transaction.note} readOnly></textarea>
```

Comment:
`note` is actually the same as `noteb64` (both are base64 values), only the name has changes so you can treat it the same as before.